### ISSUE TITLE
Failed to Sync project with Gordon v1.0.7 and AGP v3.6.0

### ISSUE THREAD

**Issue by wezley98** [Banno/Gordon#21]
Receiving the following error when syncing in Android Studio v3.6.0

- `Android Studio : com.android.tools.build:gradle:3.6.0`
- `Gradle v6.2`
- `Gordon v1.0.7`

```
ERROR: Unable to find method 'com.android.build.gradle.tasks.PackageAndroidArtifact.getOutputDirectory()Ljava/io/File;'.
Possible causes for this unexpected error include:
Gradle's dependency cache may be corrupt (this sometimes occurs after a network connection timeout.)
Re-download dependencies and sync project (requires network)

The state of a Gradle build process (daemon) may be corrupt. Stopping all Gradle daemons may solve this problem.
Stop Gradle build processes (requires restart)

Your project may be using a third-party plugin which is not compatible with the other plugins in the project or the version of Gradle requested by the project.

In the case of corrupt Gradle processes, you can also try closing the IDE and then killing all Java processes.
```

```
Could not create task ':app:gordonDebugUAT'.
> com.android.build.gradle.tasks.PackageAndroidArtifact.getOutputDirectory()Ljava/io/File;
```

Commenting out gordon fixes the issue.

**Comment by joshschriever** [comment]
Thanks for the report! Gordon hasn't been tested with Android Studio 3.6 yet. I'll get to it as soon as I can.

**Comment by joshschriever** [comment]
Fixed in #22 and released in [v1.1.0](https://github.com/Banno/Gordon/releases/tag/1.1.0)
